src/_micro-api/libraries/signalDecoder/src/signalDecoder.cpp

```diff
@@ -43,6 +43,49 @@
 #endif
 
 
+///////////////////////////////////////////////////////////////////////
+/* SELFMADE -  Implementation of itoa() */
+
+char* myitoa(int num, char* str, int base) {
+  int i = 0;
+  bool isNegative = false;
+  /* Handle 0 explicitely, otherwise empty string is printed for 0 */
+  if (num == 0) {
+    str[i++] = '0';
+    str[i] = '\0';
```

Comment:
Hier verstehe ich nicht, wieso Du '0' und anschließend '\0' in den Speicher schreibst?
Der String ist ja bereits mit '\0' terminiert.

Reply:
Ich muss gestehen, darauf kann ich dir keine Auskunft gehen, weil die Funktion selbst nach langem suchen gefunden wurde.
Das Verhalten habe ich einzeln in einem Sketch getestet und daher 
`wieso Du '0' und anschließend '\0' in den Speicher schreibst?` kann ich nur sagen, das obliegt dem Wissen *smile*

Reply:
Kommt da dann 0 oder ein leerstring zurück?

Reply:
Der String ist NICHT mit '\0' terminiert.
Wir könnten das dort ersetzen durch:

```
    str[0] = '0';
    str[1] = '\0';
```